File: Assignments/Assignment_4/program3.c
// Wrie a program which accept number from user and display all its non factors.

#include<stdio.h>

////////////////////////////////////////////////////////////////////
//
//  Function Name : PrintEven
//  Description :   It is used to display Non-factor
//  Input :         Integer
//  Output :        Integer
//  Author :        Omkar Sachin Narlawar
//  Date :          26/10/2025
//
////////////////////////////////////////////////////////////////////

void NonFact(int iNo)
{
    int iCnt = 0;

    // Updater
    if(iNo < 0)
    {
        iNo = -iNo;
    }

    for(iCnt = 1; iCnt <= iNo; iCnt++)
    {
        if((iNo % iCnt)  != 0)
        {
            printf("%d\t",iCnt);
        }
    }
}

int main()
{
    int iValue = 0;

    printf("Enter number\n");
    scanf("%d",&iValue);

    NonFact(iValue);

    return 0;
}

File: Assignments/Assignment_21/program3.java
// Write a program to display all factors of a given number.

class Logic
{
    void displayFactors(int num)
    {
        int iCnt = 0;

        for(iCnt = 1; iCnt <= num/2; iCnt++)
        {
           if(num % iCnt == 0)
           {
                System.out.println(iCnt);
           }
        }
    }
}

class program3
{
    public static void main(String agrs[])
    {
        Logic obj = new Logic();
        obj.displayFactors(12);
    }
}